SuperBass.java

```diff
@@ -8,7 +8,7 @@ public static void main(String[] args) {
         catch (InterruptedException ex)
         {
         }
-        System.out.println("This one is for the boys with the booming system") ;
+        System.out.println("This one is for the boys with the booming system");

```

Comment:
What do you think about implementing it like this:

> System.out.println(
> "This one is for the boys with the booming system"
> )
> ;

Would this be more efficient?